invenio_sequencegenerator/models.py

```diff
@@ -27,15 +27,15 @@
 from sqlalchemy.orm.exc import MultipleResultsFound
 
 from .errors import InvalidResetCall, InvalidTemplate
-from .utils import double_counter, extract_placeholders
+from .utils import double_counter, extract_placeholders, undouble_counter
 
 
 class TemplateDefinition(db.Model, object):
     """Representation of a template definition."""
```

Comment:
Could you please add more details here ? I didn't find anything in the documentation about it.